Add Tailwind CSS

hawezo opened the pull request:
<!-- Congrats on creating an Awesome list! 🎉 -->


<!-- Please fill in the below placeholders -->

https://github.com/aniftyco/awesome-tailwindcss

A curated list of awesome things related to the Tailwind CSS framework.


### By submitting this pull request I confirm I've read and complied with the below requirements 🖖

**Please read it multiple times. I spent a lot of time on these guidelines and most people miss a lot.**

## Requirements for your pull request

- **Don't waste my time.** Do a good job, adhere to all the guidelines, and be responsive.
- **You have to review at least 2 other [open pull requests](https://github.com/sindresorhus/awesome/pulls?q=is%3Apr+is%3Aopen).**
	Try to prioritize unreviewed PRs, but you can also add more comments to reviewed PRs. Go through the below list when reviewing. This requirement is meant to help make the Awesome project self-sustaining. Comment here which PRs you reviewed. You're expected to put a good effort into this and to be thorough. Look at previous PR reviews for inspiration.
- You have read and understood the [instructions for creating a list](https://github.com/sindresorhus/awesome/blob/master/create-list.md).
- This pull request has a title in the format `Add Name of List`.
	- ✅ `Add Swift`
	- ✅ `Add Software Architecture`
	- ❌ `Update readme.md`
	- ❌ `Add Awesome Swift`
	- ❌ `Add swift`
	- ❌ `Adding Swift`
	- ❌ `Added Swift`
- Your entry here should include a short description about the project/theme of the list. **It should not describe the list itself.** The first character should be uppercase and the description should end in a dot. It should be an objective description and not a tagline or marketing blurb.
	- ✅ `- [iOS](…) - Mobile operating system for Apple phones and tablets.`
	- ✅ `- [Framer](…) - Prototyping interactive UI designs.`
	- ❌ `- [iOS](…) - Resources and tools for iOS development.`
	- ❌ `- [Framer](…)`
	- ❌ `- [Framer](…) - prototyping interactive UI designs`
- Your entry should be added at the bottom of the appropriate category.
- The suggested Awesome list complies with the below requirements.


## Requirements for your Awesome list

- **Has been around for at least 30 days.**<br>That means 30 days from either the first real commit or when it was open-sourced. Whatever is most recent.
- Don't open a Draft / WIP pull request while you work on the guidelines. A pull request should be 100% ready and should adhere to all the guidelines when you open it.
- Run [`awesome-lint`](https://github.com/sindresorhus/awesome-lint) on your list and fix the reported issues. If there are false-positives or things that cannot/shouldn't be fixed, please [report it](https://github.com/sindresorhus/awesome-lint/issues/new).
- **Includes a succinct description of the project/theme at the top of the readme.** [(Example)](https://github.com/willempienaar/awesome-quantified-self)
	- ✅ `Mobile operating system for Apple phones and tablets.`
	- ✅ `Prototyping interactive UI designs.`
	- ❌ `Resources and tools for iOS development.`
	- ❌ `Awesome Framer packages and tools.`
- It's the result of hard work and the best I could possibly produce.
	**If you have not put in considerable effort into your list, your pull request will be immediately closed.**
- The repo name of your list should be in lowercase slug format: `awesome-name-of-list`.
	- ✅ `awesome-swift`
	- ✅ `awesome-web-typography`
	- ❌ `awesome-Swift`
	- ❌ `AwesomeWebTypography`
- The heading title of your list should be in [title case](https://capitalizemytitle.com/) format: `# Awesome Name of List`.
	- ✅ `# Awesome Swift`
	- ✅ `# Awesome Web Typography`
	- ❌ `# awesome-swift`
	- ❌ `# AwesomeSwift`
- Non-generated Markdown file in a GitHub repo.
- The repo should have `awesome-list` & `awesome` as [GitHub topics](https://help.github.com/articles/about-topics). I encourage you to add more relevant topics.
- Not a duplicate. Please search for existing submissions.
- Only has awesome items. Awesome lists are curations of the best, not everything.
- Does not contain items that are unmaintained, has archived repo, deprecated, or missing docs. If you really need to include such items, they should be in a separate Markdown file.
- Includes a project logo/illustration whenever possible.
	- Either centered, fullwidth, or placed at the top-right of the readme. [(Example)](https://github.com/sindresorhus/awesome-electron)
	- The image should link to the project website or any relevant website.
	- **The image should be high-DPI.** Set it to maximum half the width of the original image.
- Entries have a description, unless the title is descriptive enough by itself. It rarely is though.
- Includes the [Awesome badge](https://github.com/sindresorhus/awesome/blob/master/awesome.md#awesome-badge).
	- Should be placed on the right side of the readme heading.
		- Can be placed centered if the list has a centered graphics header.
	- Should link back to this list.
- Has a Table of Contents section.
	- Should be named `Contents`, not `Table of Contents`.
	- Should be the first section in the list.
	- Should only have one level of [nested lists](https://commonmark.org/help/tutorial/10-nestedLists.html), preferably none.
- Has an [appropriate license](https://github.com/sindresorhus/awesome/blob/master/awesome.md#choose-an-appropriate-license).
	- That means something like CC0, **not a code licence like MIT, BSD, Apache, etc.**
	- [WTFPL](http://www.wtfpl.net) and [Unlicense](http://unlicense.org) are not acceptable licenses.
	- **If you use a license badge, it should be SVG, not PNG.**
	- To verify that you've read all the guidelines, please add comment with just the word `unicorn`.
- Has [contribution guidelines](https://github.com/sindresorhus/awesome/blob/master/awesome.md#include-contribution-guidelines).
	- The file should be named `contributing.md`. Casing is up to you.
- Has consistent formatting and proper spelling/grammar.
	- The link and description are separated by a dash. <br>Example: `- [AVA](…) - JavaScript test runner.`
	- The description starts with an uppercase character and ends with a period.
	- Consistent and correct naming. For example, `Node.js`, not `NodeJS` or `node.js`.
- Doesn't include a Travis badge.<br>You can still use Travis for list linting, but the badge has no value in the readme.
- Doesn't include an `Inspired by awesome-foo` or `Inspired by the Awesome project` kinda link at the top of the readme. The Awesome badge is enough.

**Go to the top and read it again.**

joshmanders commented on readme.md:
```diff
@@ -262,6 +262,7 @@
 - [Storybook](https://github.com/lauthieb/awesome-storybook#readme) - Development environment for UI components.
 - [Blazor](https://github.com/AdrienTorris/awesome-blazor#readme) - .NET web framework using C#/Razor and HTML that runs in the browser with WebAssembly.
 - [Web Performance Metrics](https://github.com/csabapalfi/awesome-web-performance-metrics#readme) - Metrics to help understand page speed and user experience.
+- [Tailwind CSS](https://github.com/merchedhq/awesome-tailwindcss) - Utility-first CSS framework for rapid UI development.
```
This should be https://github.com/aniftyco/awesome-tailwindcss

hawezo replied:
Good catch, fixed it!

hawezo commented:
Oh, I almost forgot `unicorn`.

sindresorhus commented:
Thanks for making an Awesome list! 🙌

It looks like you didn't read the [guidelines](https://github.com/sindresorhus/awesome/blob/master/pull_request_template.md) closely enough. I noticed multiple things that are not followed. Try going through the list point for point to ensure you follow it. I spent a lot of time creating the guidelines so I wouldn't have to comment on common mistakes, and rather spend my time improving Awesome.

hawezo commented:
Hi @sindresorhus, I re-read the guidelines and I indeed missed two things.
- [x] There was no `LICENSE` file, so it wasn't detected by GitHub.
- [x] There was no logo, I added one.

Please note that `awesome-lint` has a known issue about dead links. Here is the current report after my last commit:

```console
λ awesome-lint
- Linting(node:19988) [DEP0066] DeprecationWarning: OutgoingMessage.prototype._headers is deprecated
× Linting

  readme.md:185:6
  ×  185:6  Link to https://codepen.io/joshmanders/pen/PQQBoR is dead  remark-lint:no-dead-urls
  ×  198:5  Link to https://codepen.io/adamwathan/pen/RxWrZr is dead   remark-lint:no-dead-urls
  ×  200:5  Link to https://codepen.io/drehimself/full/vpeVMx is dead  remark-lint:no-dead-urls
  ×  269:3  Link to https://hellosun.brussels is dead                  remark-lint:no-dead-urls

  4 errors
```

Those links are not actually dead.

Here is the previous checklist, manually each entry to make sure I didn't forget anything:
- [x] You have to review at least 2 other open pull requests
- [x] Read the Awesome manifesto and list guidelines and ensure your list complies.
- [x] This pull request has a title in the format Add Name of List.
- [x] Your entry here should include a short description about the project/theme of the list. It should not describe the list itself. The first character should be uppercase and the description should end in a dot. It should be an objective description and not a tagline or marketing blurb.
- [x] Your entry should be added at the bottom of the appropriate category.
- [x] Has been around for at least 30 days.
- [x] Run awesome-lint on your list and fix the reported issues. If there are false-positives or things that cannot/shouldn't be fixed, please report it. **Note**: had errors, now fixed.
- [x] Includes a succinct description of the project/theme at the top of the readme.
- [x] The repo name of your list should be in lowercase slug format: awesome-name-of-list.
- [x] The heading title of your list should be in title case format: # Awesome Name of List.
- [ ] Non-generated Markdown file in a GitHub repo.**Note** - Not sure what it means?
- [x] The repo should have awesome-list & awesome as GitHub topics. I encourage you to add more relevant topics.
- [x] Not a duplicate. Please search for existing submissions.
- [x] Only has awesome items. Awesome lists are curations of the best, not everything.
- [x] Does not contain items that are unmaintained, has archived repo, deprecated, or missing docs. If you really need to include such items, they should be in a separate Markdown file.
- [x] Includes a project logo/illustration whenever possible.
- [x] Entries have a description, unless the title is descriptive enough by itself. It rarely is though.
- [x] Includes the Awesome badge.
- [x] Has a Table of Contents section.
- [ ] Has an appropriate license.
- [x] Has contribution guidelines.
- [x] Has consistent formatting and proper spelling/grammar.
- [x] Doesn't include a Travis badge.
- [x] Doesn't include an Inspired by awesome-foo or Inspired by the Awesome project kinda link at the top of the readme. The Awesome badge is enough.

---

Thanks for your time.

sindresorhus commented:
> Allan White - Allan White's website.

This is not a useful description. It just repeats the title. It should describe who it is instead. Same applies to many of the other descriptions.

hawezo commented:
> It should describe who it is instead.

I don't think this is relevant. The descriptions was just there do differentiate blogs and websites, but this part of the list is just to showcase some good websites built with Tailwind. Describing the person about which the site is isn't necessary in my opinion.

@sindresorhus, if you still think this is an issue, tell me if I should update them anyway, or I can just remove descriptions.

sindresorhus commented:
Then you should remove the descriptions where they just repeat the title.

hawezo commented:
Alright it's done. Hope it's all good with you this time.

sindresorhus commented:
Can you linkify `Tailwind` in `Tailwind CSS is a utility-first CSS framework for rapidly building custom user interfaces.` to its website?

sindresorhus commented:
Tweet: https://twitter.com/awesome__re/status/1194225917955989515

hawezo commented:
I added the link like you suggested. Thank you! 🎉

joshmanders commented:
Thank you @sindresorhus. And especially thank you @hawezo for doing all this work!
